pages/crowdinintegration.md

```diff
@@ -1,14 +1,12 @@
 [Google Doc](https://docs.google.com/document/d/1gsDmBSq5Tu1G_Ei1CfTRUuvvwpAWUPhHYr2UYESwISI/edit?usp=sharing)
 # Crowdin Integration
 
-## Objectives:
+## Objective:
 
-There are two objectives for this assignment:
+* Provide exactly one screenshot for the strings in BeLL app, and tag it.
+* Write good description of the strings in Context.
```

Comment:
Here is the change I made (branch updated):
* Provide exactly one screenshot for the strings in BeLL app and tag it.
* Write good descriptions for the strings in Context.